Module.md

```diff
@@ -0,0 +1,189 @@
+The Azure Cosmos DB is Microsoft’s NOSQL database in Azure technology stack. It is called a globally distributed 
+multi-model database which is used for managing data across the world. Key purposes of the Azure CosmosDB is to achieve 
+low latency and high availability while maintaining a flexible scalability. 
+The Ballerina Cosmos DB connector allows you to connect to a Azure Cosmos DB resource from Ballerina and perform various 
+operations such as `find`, `create`, `read`, `update`, and `delete` operations of `Databases`, `Containers`,
+`User Defined Functions`, `Tiggers`, `Stored Procedures`, `Users`, `Permissions` and `Offers`. 
+
+## Compatibility
+
+|                           |    Version                  |
+|:-------------------------:|:---------------------------:|
+| Ballerina Language        | Swan-Lake-Preview8          |
+| Cosmos DB API Version     | 2018-12-31                  |
+
+## CosmosDB Clients
+
+There are two clients provided by Ballerina to interact with CosmosDB.
```

Comment:
```suggestion
There are two clients provided by Ballerina to interact with Cosmos DB.
```

Reply:
Check all the places

Reply:
Checked and changed Module.md, Package.md, README.md and Utils.bal